Article/Article.js

```diff
@@ -3,24 +3,36 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = "expand";
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.isOpen = false;
+    this.expandButton.addEventListener('click', () => this.expandArticle(event));
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    if(this.isOpen === false){
+      this.domElement.classList.toggle('article-open');
+      this.isOpen = true;
+      TweenMax.fromTo(".article-open", 1, { height: 50}, {height: 400} );
+    }else if(this.isOpen === true){
+      this.isOpen = false;
+      TweenMax.fromTo(".article-open", 1, { height: 400}, {height: 50} );
```

Comment:
Using `.article-open` here seems to have weird behavior when operating with multiple open articles. Maybe try using `this.domElement` for more consistent behavior? For example:
```javascript
TweenMax.to(this.domElement, 1, { height: 50 });
```